gpytorch/variational/whitened_variational_strategy.py

```diff
@@ -1,205 +0,0 @@
-#!/usr/bin/env python3
-
-import math
-import torch
-from .. import settings
-from .variational_strategy import VariationalStrategy
-from ..utils.memoize import cached
-from ..lazy import RootLazyTensor, MatmulLazyTensor, CholLazyTensor, \
-    CachedCGLazyTensor, DiagLazyTensor, BatchRepeatLazyTensor, PsdSumLazyTensor
-from ..distributions import MultivariateNormal
-
-
-class WhitenedVariationalStrategy(VariationalStrategy):
```

Comment:
Consider keeping a thin class that raises a RuntimeError explaining what happened (e.g., switch to use VariationalStrategy now, it’s whitened correctly) to deprecate.

Reply:
Ditto for any other deleted classes.

Reply:
Yeah... you're right :/
All other classes (except this one) should have deprecation warnings. I just got sloppy here.